tests/main/fakestore-install/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Ensure that the fakestore works
+
+environment:
+  BLOB_DIR: $(pwd)/fake-store-blobdir
+
+restore: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
```

Comment:
A bit of a bork-bork-bork IMO, how about `This test requires non-production root-of-trust` (or something similar)?

Reply:
Heh :) I cargo-culted this part:
```
$ git grep "This test needs test keys to be trusted"|wc -l
57
```
Probably a separate PR if we want to fix it because I think then we should fix it everywhere.